app/services/twilio_service.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class TwilioService
+
+  def initialize(cellphone, user)
+    @cellphone = cellphone
+    @user = user
+  end
+
+  def sms
+    account_sid = ENV['twilio_sid']
+    auth_token = ENV['twilio_token']
+
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    message = @client.account.messages.create(
+      from: "+17207702071",
+      to: @cellphone
```

Comment:
For the purposes of testing, we should use our real phone numbers. However, I don't know if we want our real phone numbers listed publicly. @JonathanSR, you might want to change this method so your phone number is populated by an environment variable.